src/utils/string/StringUtils.js

```diff
@@ -150,7 +150,7 @@ const displayFormattedSearchLocationsV2 = (searchQuery, locationObject) => {
       return (
         <Text>
           <TextBold text={`${neighborhoodDisplay}`} />
-          {`${cityDisplay}${stateDisplay}`}
+          <Text style={{color: COLORS.gray500}}>{`${cityDisplay}${stateDisplay}`}</Text>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change introduces a hardcoded color value `COLORS.gray500`. It's better to use a centralized approach for managing colors in your application. This will make it easier to maintain and update the color scheme of your app. Consider using a global style or theme.

```diff
- <Text style={{color: COLORS.gray500}}>{`${cityDisplay}${stateDisplay}`}</Text>
+ <Text style={styles.cityStateDisplay}>{`${cityDisplay}${stateDisplay}`}</Text>
```
In your styles.js file, add:

```javascript
cityStateDisplay: {
  color: COLORS.gray500,
},
```
This way, if you ever need to change the color, you only have to do it in one place.


<!-- This is an auto-generated comment by OSS CodeRabbit -->